tszip/cli.py

```diff
@@ -102,6 +108,8 @@ def check_output(outfile, args):
 
 
 def run_compress(args):
+    if args.stdout:
+        exit("Cannot compress from stdout currently.")
```

Comment:
Should probably be compress *to* stdout? Also would be nice to tell people where to go if they want to see more info. How about,

```
exit(
    "Compressing to stdout not currently supported; "
    "please see https://github.com/tskit-dev/tszip/issues/49 for more information."
)
```
```

Reply:
Great solution - I think that the second line is too long for `pre-commit` so I've amended it to:
`"please see https://github.com/tskit-dev/tszip/issues/49"`.